chia/full_node/bundle_tools.py

```diff
@@ -88,7 +92,7 @@ def bundle_suitable_for_compression(bundle: SpendBundle):
 
 
 def compressed_coin_solution_entry_list(bundle: SpendBundle) -> List:
-    compressed_cse_list = []
+    compressed_cse_list: List[List[Union[List[uint64], List[Union[bytes, None, Program]]]]] = []
```

Comment:
This type annotation made me second guess this output type when I was writing it. I guess I should add a new struct type for this.